parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -154,7 +154,7 @@ time_metric "InstallNetworkPlugin" installNetworkPlugin
 
 {{- if and HasNSeriesSKU IsNvidiaDevicePluginAddonEnabled}}
 if [[ ${GPU_NODE} == true ]]; then
-  time_metric "DownloadGPUDrivers" downloadGPUDrivers
+  gpuDriversDownloaded || time_metric "DownloadGPUDrivers" downloadGPUDrivers
```

Comment:
after a successful install, should the permanent cache be deleted to free up space?  (Not significant but it is space that could otherwise be used for container caches)

This is non-blocking question - we can ignore that

Reply:
done